MultimodalQnA/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -1,139 +1,99 @@
-
-  ```bash
-  git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout v1.3
-  ```
-
-  We remind you that when using a specific version of the code, you need to use the README from this version.
+```bash
+git clone https://github.com/opea-project/GenAIExamples.git
+cd GenAIExamples/MultimodalQnA
+```
 
-- #### Setting the list of images for the build (from the build file.yaml)
+Then checkout a released version, such as v1.3:
 
-  If you want to deploy a vLLM-based or TGI-based application, then the set of services is installed as follows:
+```bash
+git checkout v1.3
+```
 
-  #### vLLM-based application
+### Configure the Deployment Environment
 
-  ```bash
-  service_list="multimodalqna multimodalqna-ui embedding-multimodal-bridgetower embedding retriever lvm dataprep whisper vllm-rocm"
-  ```
+To set up environment variables for deploying MultimodalQnA services, set up some parameters specific to the deployment environment and source the `set_env_*.sh` script in this directory:
 
-  #### TGI-based application
+- if used vLLM - set_env_vllm.sh
+- if used TGI - set_env.sh
 
-  ```bash
-  service_list="multimodalqna multimodalqna-ui embedding-multimodal-bridgetower embedding retriever lvm dataprep whisper"
-  ```
+Set the values of the variables:
 
-- #### Optional. Pull TGI Docker Image (Do this if you want to use TGI)
+- **HOST_IP, HOST_IP_EXTERNAL** - These variables are used to configure the name/address of the service in the operating system environment for the application services to interact with each other and with the outside world.
 
-  ```bash
-  docker pull ghcr.io/huggingface/text-generation-inference:2.3.1-rocm
-  ```
+  If your server uses only an internal address and is not accessible from the Internet, then the values for these two variables will be the same and the value will be equal to the server's internal name/address.
 
-- #### Build Docker Images
+  If your server uses only an external, Internet-accessible address, then the values for these two variables will be the same and the value will be equal to the server's external name/address.
 
-  ```bash
-  docker compose -f build.yaml build ${service_list} --no-cache
-  ```
+  If your server is located on an internal network, has an internal address, but is accessible from the Internet via a proxy/firewall/load balancer, then the HOST_IP variable will have a value equal to the internal name/address of the server, and the EXTERNAL_HOST_IP variable will have a value equal to the external name/address of the proxy/firewall/load balancer behind which the server is located.
 
-  After the build, we check the list of images with the command:
+  We set these values in the file set_env\*\*\*\*.sh
 
-  ```bash
-  docker image ls
-  ```
+- **Variables with names like "**\*\*\*\*\*\*\_PORT"\*\* - These variables set the IP port numbers for establishing network connections to the application services.
```

Comment:
Same with this one:
`"**\*\*\*\*\*\*\_PORT"\*\*` -> `"*\*\*\_PORT\*\*\*"`
Unless I'm missing something.